bundles/org.eclipse.swt/Eclipse SWT/cocoa/org/eclipse/swt/graphics/Image.java

```diff
@@ -527,7 +527,11 @@ private void createRepFromSourceAndApplyFlag(NSBitmapImageRep srcRep, int srcWid
  * </ul>
  *
  * @see #dispose()
+ *
+ * @deprecated use {@link Image#Image(Device, int, int)} instead
+ * @since 3.130
  */
+@Deprecated(since = "2025-05", forRemoval = true)
```

Comment:
The release is 2025-06
```suggestion
@Deprecated(since = "2025-06", forRemoval = true)
```